manifest.yml

```diff
@@ -9,6 +9,9 @@ applications:
     HSES_AUTH_BASE: ((hses_auth_base))
     HSES_AUTH_CLIENT_ID: ((hses_auth_client_id))
     HSES_AUTH_CLIENT_SECRET: ((hses_auth_client_secret))
+    HSES_API_BASE: ((hses_api_base))
+    HSES_API_USERNAME: ((hses_api_username))
+    HSES_API_PASSWORD: ((hses_api_password))
```

Comment:
[suggestion] Add lines to `config/deployment/stage.yml` for `hses_api_base` and `hses_api_username` and the appropriate env var and `push_arguments` to `.github/workflows/deploy-stage.yml` for the password to be grabbed from the staging secrets

Reply:
Good catch. Guess I gotta delete a couple base / username secrets that I stuck into github when I got too secret happy.